packages/hardhat-solhint/test/fixture-projects/no-errors-project/contracts/Greeter.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.8.0;
```

Comment:
Do we know why the `0.8.0` version was needed rather than the other examples that use `0.5`?

Reply:
This is a default minimal compiler version set by solhint (see: https://github.com/protofire/solhint/blob/v5.0.2/docs/rules/security/compiler-version.md#compiler-version)

```suggestion
// NOTE: This is a default minimal compiler version set by solhint (see: https://github.com/protofire/solhint/blob/v5.0.2/docs/rules/security/compiler-version.md#compiler-version)
pragma solidity ^0.8.0;
```